scripts/run_e2e_tests_test.py

```diff
@@ -40,14 +41,25 @@
 
 
 class MockProcessClass(python_utils.OBJECT):
-    def __init__(self):
-        pass
-
-    kill_count = 0
+    def __init__(self, clean_shutdown=True):
+        self.poll_count = 0
+        self.signals_received = []
+        self.kill_count = 0
+        self.poll_return = True
+        self.clean_shutdown = clean_shutdown
 
     # pylint: disable=missing-docstring
```

Comment:
Probably about time we added docstrings.

Reply:
done in 97bba1a74